.dev/bin/setup-test-specs.sh

```diff
@@ -27,9 +27,13 @@ do
   if [[ $FILE == *"src/extensions/"* ]]; then
     testSpec=$(echo $FILE | cut -d'/' -f3)
 		foundwords=$(echo ${SPECS[@]} | grep -o "${testSpec}" | wc -w)
+		# Catch cases where no cypress spec file exists. 
+			if [[ $(ls -l src/extensions/${testSpec}/**/*.cypress.js | wc -l) -eq 0 ]]; then
```

Comment:
@AnthonyLedesma These lines can probably be brought back one tab, so it's inline with `foundwords` on the line above.

Reply:
Good call! Fixed.